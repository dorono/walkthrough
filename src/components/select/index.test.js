import React from 'react';
import {shallow} from 'enzyme';
import Select from './index';

const defaultProps = {
    options: [],
    value: '',
    onChange: () => null,
};

describe('Select', () => {
    it('should render without errors', () => {
        shallow(<Select {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<Select {...defaultProps} />)).toMatchSnapshot();
    });
});
