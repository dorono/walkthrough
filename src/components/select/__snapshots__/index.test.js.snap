// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select should match snapshot 1`] = `
<div
  className="root"
>
  <div
    className="value"
  />
  <select
    className="select"
    onChange={[Function]}
    value=""
  />
</div>
`;
